desktop/extensions-sdk/dev/continuous-integration.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Continuous Integration"
+description: Automatically test and validate your extension.
+keywords: Docker, Extensions, sdk, CI, test, regression
+---
+
+In order to help validating your extension and ensure it's functional, the Extension SDK provides a set of tools to help you set continuous integration for your extension.
+
+> The github action and the extension-test-helper library are both [experimental](https://docs.docker.com/release-lifecycle/#experimental).
```

Comment:
Is there a link for more info on the extension-test-helper library?

Reply:
Good catch, I added both links here (the GHA link was already mentioned below)